### ISSUE TITLE
textarea appears to only work with mouse

### ISSUE THREAD

**Issue by DavidMacDonald** [w3c/html#605]
The `<textarea>` element renders in browsers with the familiar triangle in bottom right of the text box to drag to resize but there is no keyboard equivalent. This is a failure of WCAG 2 SC 2.1.1 Keyboard requirements and forces authors to do complicated things to resize with keyboard. This functionality should be natively available.
https://www.w3.org/TR/UNDERSTANDING-WCAG20/keyboard-operation-keyboard-operable.html

**Comment by chaals** [comment]
Hi David,

This seems primarily a browser bug, not a problem with the specification. However, I think it makes sense to note in the specification that the ability to resize - like other interactions - has accessibility implications.

At the same time, filing browser bugs seems the most appropriate path here.

**Comment by stevefaulkner** [comment]
filed bug on Firefox https://bugzilla.mozilla.org/show_bug.cgi?id=1307520
and chrome https://bugs.chromium.org/p/chromium/issues/detail?id=652753&can=2&q=textarea%20resize%20keyboard&colspec=ID%20Pri%20M%20Stars%20ReleaseBlock%20Component%20Status%20Owner%20Summary%20OS%20Modified

FYI
Doesn't look like edge or Safari/webkit on iOS at least, provides the same feature for textarea

**Comment by LJWatson** [comment]
Thanks for filing this @DavidMacDonald, and @stevefaulkner for filing browser issues.
